shared-module/displayio/Bitmap.c

```diff
@@ -104,6 +109,26 @@ void common_hal_displayio_bitmap_set_pixel(displayio_bitmap_t *self, int16_t x,
     if (self->read_only) {
         mp_raise_RuntimeError(translate("Read-only object"));
     }
+    // Update the dirty area.
+    if (self->dirty_area.x1 == self->dirty_area.x2) {
+        self->dirty_area.x1 = x;
+        self->dirty_area.x2 = x + 1;
```

Comment:
Is it ok if .x2 or .y2 are out of bounds with respect to the bitmap? You don't do a `min(x +1, size_x)` kind of range limiting.

Reply:
The range limiting is done in shared-bindings so it'll be fine here.